lib/tasks/convert_salary_ranges_to_salary_string.rake

```diff
@@ -0,0 +1,24 @@
+namespace :data do
+  desc 'Convert salary ranges to string'
+  namespace :convert_salary_ranges do
+    task vacancies: :environment do
+      Vacancy.in_batches.each_record do |vacancy|
+        min_pay_scale = vacancy.min_pay_scale_id.present? ? PayScale.find(vacancy.min_pay_scale_id) : nil
```

Comment:
So, first off, I'd get rid of these two db calls - they're comparatively expensive and you've already got the data in `PAYSCALE_DATA` in `seeds.rb`.   Just move it to it's own initializer and use that everywhere you need it.